### PR TITLE
Documenting Lessons Learnt from recent CVE announcement

### PR DESCRIPTION
**Pull request by PushkarJ**
- Clarified language on announcement of medium/low CVEs with public fixes
- Updated the CVSS calculator from 3.1 to 3.0
- Added useful tips in comms. template for vulnerability announcement email
- Change kubernetes-dev to dev (at) kubernetes.io in onboarding checklist
- Fixed broken link for upgrading a cluster

/cc @cjcullen @tabbysable @tallclair @ritazh @enj

### CODE REVIEW

File: comms-templates/vulnerability-announcement-email.md [hunk 1]
```diff
@@ -7,12 +7,17 @@ issue](vulnerability-announcement-issue.md)._
 
 TO: `kubernetes-announce@googlegroups.com, dev@kubernetes.io, kubernetes-security-announce@googlegroups.com, kubernetes-security-discuss@googlegroups.com, distributors-announce@kubernetes.io, kubernetes+announcements@discoursemail.com`
 
+Tip: Make sure you are a member / manager of these groups before sending the email as posting new topics is restricted
+to members.
```
Comment (ritazh) [inline comms-templates/vulnerability-announcement-email.md:11]: who do we contact to be added?

File: security-release-process.md [hunk 2]
```diff
@@ -216,6 +216,8 @@ Communications process:
     [patch release](https://github.com/kubernetes/sig-release/blob/master/releases/patch-releases.md)
     will have the fix details included in the patch release notes. Any public announcement sent for
     these fixes will link to the release notes.
+  - Medium and Low vulnerabilities with public fixes (PRs) can be announced on private
+    distributors list at the same time and as part of the general announcement email.
```
[the remark this answers is not in the document]
Comment (PushkarJ) [reply security-release-process.md:220]: I purposely used this language to clarify that we don't need a separate announcement for distributors and general community as both can be recipients of the _same_ email announcement but sounds like I may not have done a great job with the current wording to explain that. Could you suggest a change that clarifies this better ?

[the remark this answers is not in the document]
Comment (ritazh) [reply security-release-process.md:220]: What do you think about my suggestion above?

File: comms-templates/vulnerability-announcement-email.md (hunk 1 above)
Comment (SaranBalaji90) [reply comms-templates/vulnerability-announcement-email.md:11]: We can add ourself by just clicking "JoinList" to be a member.

File: security-release-process.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (SaranBalaji90) [reply security-release-process.md:220]: Do we only send embargo issues to distributors list? I'm wondering why not send the announcement to distributor list one week before itself for all CVEs to have them assess and re-score the CVE if needed considering their platform.

File: comms-templates/vulnerability-announcement-email.md (hunk 1 above)
Comment (joelsmith) [reply comms-templates/vulnerability-announcement-email.md:11]: During the onboarding process the existing SRC members will make you an owner of some groups and a manager of another, but the dev@ list requires you to join it yourself.

File: security-release-process.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (ritazh) [reply security-release-process.md:220]: @PushkarJ Can you please take a look at the suggestion so we can merge this PR?

File: comms-templates/vulnerability-announcement-email.md [hunk 3]
```diff
@@ -7,12 +7,17 @@ issue](vulnerability-announcement-issue.md)._
 
 TO: `kubernetes-announce@googlegroups.com, dev@kubernetes.io, kubernetes-security-announce@googlegroups.com, kubernetes-security-discuss@googlegroups.com, distributors-announce@kubernetes.io, kubernetes+announcements@discoursemail.com`
 
+Tip: Make sure you are a member / manager of these groups before sending the email as posting new topics is restricted
```
Comment (PushkarJ) [inline comms-templates/vulnerability-announcement-email.md:10]:
```suggestion
Tip: Make sure you are a member / manager of these groups before sending the email as posting new topics is restricted. Ask your fellow SRC members on how to get yourself added to these groups. 
```

**Comment by PushkarJ** [comment]
/remove-lifecycle rotten

The pending edits should now be resolved. It took me a while but hopefully this is now ready to merge.

**Comment by ritazh** [comment]
/lgtm

@SaranBalaji90 ready to merge?

File: comms-templates/vulnerability-announcement-email.md [hunk 4]
```diff
@@ -7,12 +7,17 @@ issue](vulnerability-announcement-issue.md)._
 
 TO: `kubernetes-announce@googlegroups.com, dev@kubernetes.io, kubernetes-security-announce@googlegroups.com, kubernetes-security-discuss@googlegroups.com, distributors-announce@kubernetes.io, kubernetes+announcements@discoursemail.com`
 
+Tip: Make sure you are a member / manager of these groups before sending the email as posting new topics is restricted. Ask your fellow SRC members on how to get yourself added to these groups.
```
Comment (tallclair) [inline comms-templates/vulnerability-announcement-email.md:10]: nit: can you format these a little differently to make them stand out from the other content a bit? Maybe quote them?
```suggestion
> Tip: Make sure you are a member / manager of these groups before sending the email as posting new topics is restricted. Ask your fellow SRC members on how to get yourself added to these groups. 
```

**Review by tallclair — COMMENTED** [review]
/lgtm

**Comment by SaranBalaji90** [comment]
@PushkarJ thank you for the PR. Let me know if you have bandwidth to resolve conflicts. If not I'm happy to help.

**Comment by PushkarJ** [comment]
@SaranBalaji90 I am okay for you to take it forward if you can!

**Pull request closed without merging**